demisto_sdk/commands/common/hook_validations/docker.py

```diff
@@ -180,10 +174,11 @@ def find_latest_tag_by_date(tags):
         latest_tag_date = datetime.now() - timedelta(days=400000)
         for tag in tags:
             tag_date = datetime.strptime(tag.get('last_updated'), '%Y-%m-%dT%H:%M:%S.%fZ')
-            if tag_date >= latest_tag_date:
+            if tag_date >= latest_tag_date and tag.get('name') != 'latest':
                 latest_tag_date = tag_date
                 latest_tag_name = tag.get('name')
-
+        if latest_tag_name == 'latest':
```

Comment:
not sure can this condition be true, you're checking in the previous condition for that..

Reply:
It's true only in case there is only 1 docker image in the docker-hub and it labelled "latest" (I saw that currently there are couple of those)